BUILD

```diff
@@ -127,6 +127,125 @@ tfrt_cc_binary(
     ],
 )
 
+tfrt_cc_library(
+    name = "one_hot_encoder_cc",
```

Comment:
Creating one bazel rule for every algorithm may unnecessary. Could we just add these files to the existing `clink_kernels` as the umbrella rule for now?

In the long term we could follow the way of how TFRT groups kernels into rules.

We could move infra related classes (e.g. `sparse_vector.cc`) to a dedicate rule.

Reply:
OK. I'll make the change.